configs/testnet.json

```diff
@@ -103,6 +103,12 @@
       "Mantle Testnet": 5003,
       "OP Sepolia Testnet": 11155420,
       "Monad Testnet": 10143
+    },
+    "override_evm_chain_name_map": {
+      "BSC Testnet": "Binance Smart Chain",
+      "Mantle Testnet": "Mantle",
+      "OP Sepolia Testnet": "Optimism",
+      "Monad Testnet": "Monad"
     }
```

Comment:
Pls add the definition of this field into `config.schema.json` and `.github\markets\pr_template.md`